auctioneer/client.go

```diff
@@ -1318,6 +1321,108 @@ func (c *Client) MarketInfo(ctx context.Context) (
 	return c.client.MarketInfo(ctx, &auctioneerrpc.MarketInfoRequest{})
 }
 
+// InitAccountCipherBox attempts to initialize a new CipherBox using the
+// sidecar ticket as the authentication method.
+func (c *Client) InitTicketCipherBox(ctx context.Context, sid [64]byte,
+	ticket *sidecar.Ticket) error {
+
+	// TODO(roasbeef): add error to catch deupliacte stream
+	// existence/creation, also need to allow stream deletion as well
+
+	strTicket, err := sidecar.EncodeToString(ticket)
+	if err != nil {
+		return err
+	}
+
+	streamInit := &auctioneerrpc.CipherBoxInit{
+		Desc: &auctioneerrpc.CipherBoxDesc{
+			StreamId: sid[:],
+		},
+		Auth: &auctioneerrpc.CipherBoxInit_SidecarAuth{
+			SidecarAuth: &auctioneerrpc.SidecarAuth{
+				Ticket: strTicket,
+			},
+		},
+	}
+	_, err = c.hashMailClient.NewCipherBox(ctx, streamInit)
+	return err
+}
+
+// InitAccountCipherBox attempts to initialize a new CipherBox using the
+// account key as an authentication mechanism.
+func (c *Client) InitAccountCipherBox(ctx context.Context, sid [64]byte,
+	acctKey *keychain.KeyDescriptor) error {
+
+	streamSig, err := c.cfg.Signer.SignMessage(
+		ctx, sid[:], acctKey.KeyLocator,
```

Comment:
should we add a prefix so we are not just signing arbitrary data here?

Reply:
IIRC this call ends up adding the prefix underneath by default.

Reply:
Ah, true